examples/fluids/Makefile

```diff
@@ -36,7 +38,16 @@ OBJDIR := build
 SRCDIR := src
 PROBLEMDIR := problems
 
-src.c := navierstokes.c $(sort $(wildcard $(PROBLEMDIR)/*.c)) $(sort $(wildcard $(SRCDIR)/*.c))
+src.c := navierstokes.c $(filter-out $(PROBLEMDIR)/%-ad.c, $(sort $(wildcard $(PROBLEMDIR)/*.c))) $(sort $(wildcard $(SRCDIR)/*.c))
+
+# Enzyme-AD
+ENZYME_LIB ?=
+ENZYMEFLAGS = -Xclang -load -Xclang $(ENZYME_LIB) -O2 -fno-unroll-loops -mllvm -enzyme-print
```

Comment:
You can likely remove the `-fno-unroll-loops -mllvm -enzyme-print`

Reply:
Good; yes, best to be minimally intrusive.

@wsmoses do you think it's feasible to use Enzyme within hiprtc? I think this would require building with the LLVM behind `hipcc`, or perhaps just an LLVM from the same LLVM major version (and expect it to be ABI compatible). We're starting to run on OLCF Crusher (same arch as Frontier) and it would be wonderful if we could use Enzyme there. (We can move to an issue in the Enzyme repo if that's a better place to discuss.)

Reply:
We've used Enzyme on AMD kernels via Julia, and I don't see a reason that shouldn't work for hipcc (since indeed we should be compatible across LLVM 7-current). Frankly the only limiting factor is a lack of compatible ROCm-compatible GPU to try it out on. I can probably ask @jdoerfert to lend me one on ALCF.

Reply:
If you email me your ssh public key, I can add you to our development box (Radeon VII -- a little old, but still useful for testing). We can probably also get you access to OLCF via ECP -- that takes longer because they mail you a token and you have to mail back a notarized form. If you have JLSE access (https://www.jlse.anl.gov/hardware-under-development/), that would also work.

Reply:
Thanks, @wsmoses ! Just removed those flags.